scripts/RT/sweep_worker.py

```
@@ -36,20 +37,18 @@ def drain(ch, delivery_tag, _payload):
     ch.connection.add_callback_threadsafe(cb)
 
 
-def run_command(cmd: list[str]) -> bool:
+def run_command(cmd: list[str], tempdir: str) -> bool:
     """Common command running logic."""
     with subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)